dataprofiler/profilers/profile_builder.py

```diff
@@ -2290,7 +2295,12 @@ def _update_null_replication_metrics(self, clean_samples: Dict) -> None:
             sum_not_null = np.delete(total_row_sum, col_id) - sum_null
 
             mean_null = sum_null / null_count
-            mean_not_null = sum_not_null / true_count
+
+            mean_not_null = np.full(len(self._profile) - 1, np.nan)
+            if not true_count:
+                sum_not_null = np.full(len(self._profile) - 1, np.nan)
```

Comment:
if `true_count` is zero (`if not true_count`)...which sets `sum_not_null` to a single array ... not an `array` of `arrays` like `sum_not_null` on L2295 returns...